WalletWasabi.Fluent/ViewModels/Wallets/WalletSettingsViewModel.cs

```diff
@@ -84,4 +114,27 @@ public WalletSettingsViewModel(WalletViewModelBase walletViewModelBase)
 	public override sealed string Title { get; protected set; }
 
 	public ICommand VerifyRecoveryWordsCommand { get; }
+
+	private void ValidatePlebStopThreshold(IValidationErrors errors) =>
+		ValidatePlebStopThreshold(errors, PlebStopThreshold);
+
+	private static void ValidatePlebStopThreshold(IValidationErrors errors, string plebStopThreshold)
+	{
+		if (string.IsNullOrWhiteSpace(plebStopThreshold))
+		{
+			return;
+		}
+
+		if (!string.IsNullOrEmpty(plebStopThreshold) && plebStopThreshold.Contains(
+			',',
+			StringComparison.InvariantCultureIgnoreCase))
+		{
+			errors.Add(ErrorSeverity.Error, "Use decimal point instead of comma.");
```

Comment:
add `return;` after `Add`.

Reply:
How about just using `else if` to not add these `return;`?